.github/workflows/sphinx.yml

```diff
@@ -5,7 +5,9 @@ name: Sphinx build
 
 on:
   push:
-    branches: [ main ]
+    branches: [ docs-build ]
```

Comment:
What does this do, is it so there's a special branch to which we can push to force doc rebuilds, but also docs will be rebuilt when a new tag is created?

Reply:
Yeah, I figured if there's a typo in the docs or something like that, we should have a way to update that in between tags. There's a protected branch created for that.